cmd/apiserver_receive_adapter/main.go

```diff
@@ -63,46 +59,33 @@ func (s *StringList) Decode(value string) error {
 }
 
 type envConfig struct {
-	Namespace       string     `envconfig:"SYSTEM_NAMESPACE" default:"default"`
+	adapter.EnvConfig
+
 	Mode            string     `envconfig:"MODE"`
-	SinkURI         string     `split_words:"true" required:"true"`
 	ApiVersion      StringList `split_words:"true" required:"true"`
 	Kind            StringList `required:"true"`
 	Controller      []bool     `required:"true"`
 	LabelSelector   StringList `envconfig:"SELECTOR" required:"true"`
 	OwnerApiVersion StringList `envconfig:"OWNER_API_VERSION" required:"true"`
 	OwnerKind       StringList `envconfig:"OWNER_KIND" required:"true"`
 	Name            string     `envconfig:"NAME" required:"true"`
-	// MetricsConfigJson is a json string of metrics.ExporterOptions.
-	// This is used to configure the metrics exporter options,
-	// the config is stored in a config map inside the controllers
-	// namespace and copied here.
-	MetricsConfigJson string `envconfig:"K_METRICS_CONFIG" required:"true"`
-
-	// LoggingConfigJson is a json string of logging.Config.
-	// This is used to configure the logging config, the config is stored in
-	// a config map inside the controllers namespace and copied here.
-	LoggingConfigJson string `envconfig:"K_LOGGING_CONFIG" required:"true"`
 }
 
-// TODO: the controller should take the list of GVR
```

Comment:
It looks like we lost this TODO; it looks like there's a dynamic list in the current code. I'm not sure what the original intent of this TODO was, though. Maybe it's obsolete?

Reply:
@n3wscott , who left this comment in #1175

Reply:
This is being tracked here https://github.com/knative/eventing/issues/1660 instead of the TODO. Is that alright?